cvat-core/src/enums.ts

```diff
@@ -50,6 +50,7 @@ export enum RQStatus {
     FINISHED = 'finished',
     FAILED = 'failed',
     UNKNOWN = 'unknown',
+    DEFERRED = 'deferred',
```

Comment:
Did you test it?

I have several deferred requests, but they are returned as queued
![image](https://github.com/cvat-ai/cvat/assets/40690378/7ce84dee-a822-403c-961b-70449a5c9bad)

Reply:
> I have several deferred requests, but they are returned as queued

This is consciously changed on the server, a user shouldn't know that his job is deferred. For him job is just in a queue.

Reply:
> a user shouldn't know that his job is deferred

1. Why do you think so? 
2. For a user on UI we will still may show "Queued", no problem.
3. But this information is required to implement correct long-time-pooling algorithm  (when user has 500+ jobs queued, will client ask the server about each status every N seconds??? Definitely No -> throttling issues. So, for which Requests should client fetch the information? I would say that we not even should now which jobs are deferred [at minimum], but also information about dependend jobs is important)

Reply:
Okay, let's postpone this algorithm, but in this case please remove all 'deferred' from cvat-core